### PR TITLE
GPT-112 - Custom Data Tab

### PR DESCRIPTION
**Pull request by michaelsexton**
Add KML File and URL to custom data tab, add text description of WMS URL, and remove fixed width for the WMS URL text field.

### CODE REVIEW

File: src/main/webapp/js/auscope/widgets/CustomRecordPanel.js [hunk 1]
```diff
@@ -163,75 +64,54 @@ Ext.define('auscope.widgets.CustomRecordPanel', {
...
+	                name: 'url',
+	                fieldLabel: 'KML URL',
+	                labelWidth: 60,
+	                msgTarget: 'side',
+	                allowBlank: false,
+	                anchor: '100%'
+			},
+			{xtype:'panel',
+				layout: 'table',
+				html: 'Blablablabla'}],
```
Comment (heyagosper) [inline src/main/webapp/js/auscope/widgets/CustomRecordPanel.js]: what's this bit for? Just layout? if so there is an ExtJS spacer component you can use instead.

File: src/main/webapp/js/auscope/widgets/CustomRecordPanel.js [hunk 2]
```diff
@@ -18,30 +18,9 @@ Ext.define('auscope.widgets.CustomRecordPanel', {
...
-                    xtype:'tbseparator'
-                },{
-                    xtype : 'button',
-                    text      : 'More',
-                    iconCls    :   'setting fa-spin',
-                    arrowAlign: 'right',
-                    menu      : menuItems
-                }]
-            });
+        	this.addDocked(this._getCustomWMS());
```
Comment (heyagosper) [inline src/main/webapp/js/auscope/widgets/CustomRecordPanel.js]: stray tab here

File: src/main/webapp/js/auscope/widgets/CustomRecordPanel.js [hunk 3]
```diff
@@ -18,30 +18,9 @@ Ext.define('auscope.widgets.CustomRecordPanel', {
```
Comment (heyagosper) [inline src/main/webapp/js/auscope/widgets/CustomRecordPanel.js:18]: there are a few blank lines and related formatting stuff in this file. Could you tidy it up a bit in this work (I know, it wasn't written by you but here is a chance to do some code cleanup)

File: src/main/webapp/js/auscope/widgets/CustomRecordPanel.js (hunk 1 above)
Comment (michaelsexton) [inline src/main/webapp/js/auscope/widgets/CustomRecordPanel.js]: I don't even remember writing this code. Will fix.

**Comment by michaelsexton** [comment]
@heyagosper plz check.

**Comment by heyagosper** [comment]
Looks like you need to set eclipse to use 4 spaces instead of tabs...

**Pull request merged**